### PR TITLE
Apply immediate blacklisting immediately.

### PR DESCRIPTION
**Pull request by hammerhead**

### CODE REVIEW

File: lib/soapy_cake/admin.rb [hunk 1]
```diff
@@ -109,11 +109,16 @@ def update_campaign(opts = {})
     def add_blacklist(opts)
       require_params(opts, %i(blacklist_date))
 
+      blacklist_date = opts[:blacklist_date]
```
Comment (dziemba) [inline lib/soapy_cake/admin.rb]: What if this is a `DateTime`? Also, the api doc says it has to be in the format `MM-DD-YYYY`, that's probably not what `Date#to_s` does, right?

See: https://support.getcake.com/support/solutions/articles/5000593922-addedit-blacklist-api-version-1

Comment (hammerhead) [inline lib/soapy_cake/admin.rb]: We can add a 'to_date' to be sure. The format should be fine, used to be like that all the time.

**Comment by dziemba** [comment]
This probably needs fixing then: https://github.com/ad2games/backoffice/blob/master/app/models/blacklist.rb#L55

**Comment by hammerhead** [comment]
https://github.com/ad2games/backoffice/pull/418

**Comment by dziemba** [comment]
Right... Didn't see that one ;)

**Comment by hammerhead** [comment]
So I consider this :mushroom:-ed, if not, just let me know...

**Comment by dziemba** [comment]
Looks good ;)

**Pull request merged**